dist/redhat/scylla-jmx.spec.mustache

```diff
@@ -41,6 +41,7 @@ fi
 
 %post
 %systemd_post scylla-jmx.service
+/usr/bin/systemctl daemon-reload
```

Comment:
Will this fail in docker where systemd is not installed?

Reply:
Should have `|| :` at the end to suppress errors. https://docs.pagure.org/packaging-guidelines/Packaging%3AScriptlets.html

Reply:
Updated the code: https://github.com/scylladb/scylla-jmx/pull/92/files#diff-05421edf403fa53414100044a254258eR44